browser/mapping.go

```diff
@@ -457,6 +465,39 @@ func mapFrame(vu moduleVU, f *common.Frame) mapping {
 	return maps
 }
 
+func parseWaitForFunctionArgs(
```

Comment:
Not sure about this. Should i just copy and paste the parsing instead of trying to abstract them out into its own function?

Reply:
I prefer to keep it this way, grouped in a separate function, instead of having it scattered around the mappings implementation. Also considering that this is called more than once, for `mapPage` and `mapFrame`.